### PR TITLE
sub-menu-admin

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

File: app/views/layouts/_admin_header.html.erb [hunk 1]
```diff
@@ -69,16 +69,53 @@
...
+          <ul class="treeview-menu">
+            <li><a href="/admin/ranks"><i class="fa fa-circle-o"></i><%= t "layouts.ranks" %></a></li>
+          </ul>
         </li>
-        <li>
-          <a href="/admin/profiles">
-            <i class="fa fa-calendar"></i><span><%= t("layouts.user_profiles") %></span>
+        <li class="treeview">
+          <a href="#">
+            <i class="fa fa-book"></i><span><%= t("layouts.user_profiles") %></span>
```
Comment (truonglocbinh) [inline app/views/layouts/_admin_header.html.erb]: remove ()

Comment (truonglocbinh) [inline app/views/layouts/_admin_header.html.erb]: remove ()

File: app/views/layouts/_admin_header.html.erb [hunk 2]
```diff
@@ -69,16 +69,53 @@
             <i class="fa fa-table"></i><span><%= t("layouts.roles") %></span>
           </a>
         </li>
-        <li>
-          <a href="/admin/evaluations">
+        <li class="treeview">
+          <a href="#">
             <i class="fa fa-book"></i><span><%= t("layouts.evaluations") %></span>
```
Comment (truonglocbinh) [inline app/views/layouts/_admin_header.html.erb]: remove ()

**Comment by truonglocbinh** [comment]
ok

**Comment by NgengChhengkim** [comment]
OK

**Comment by thiennv57** [comment]
ok

**Comment by vuducluan** [comment]
OK

**Pull request merged**